ddtrace/ext/test.py

```diff
@@ -5,35 +5,35 @@
 from enum import Enum
 
 
-# Test Arguments
-ARGUMENTS = TEST_ARGUMENTS = "test.arguments"
-
 # Test Framework
-FRAMEWORK = TEST_FRAMEWORK = "test.framework"
```

Comment:
why are we dropping all of these names?

Is there any chance anyone is using these manually?

Reply:
They also seem unrelated to this PR so I'd be in favour of leaving these changes out

Reply:
I wasn't sure why we had duplicate names for the same constants in the first place. I can deprecate them instead, does that work?